tests/acceptance/view-user-profile-test.js

```
@@ -19,6 +19,21 @@ module('Acceptance | view-user-profile', function (hooks) {
+    let cpp = this.server.schema.languages.findBy({ slug: 'cpp' });
+    let java = this.server.schema.languages.findBy({ slug: 'java' });
+    let javascript = this.server.schema.languages.findBy({ slug: 'javascript' });
+    let haskell = this.server.schema.languages.findBy({ slug: 'haskell' });
+    let c = this.server.schema.languages.findBy({ slug: 'c' });
+    let rust = this.server.schema.languages.findBy({ slug: 'rust' });
+    let ruby = this.server.schema.languages.findBy({ slug: 'ruby' });
+    let php = this.server.schema.languages.findBy({ slug: 'php' });
+    let swift = this.server.schema.languages.findBy({ slug: 'swift' });
+    let typescript = this.server.schema.languages.findBy({ slug: 'typescript' });
+    let kotlin = this.server.schema.languages.findBy({ slug: 'kotlin' });
+    let objectivec = this.server.schema.languages.findBy({ slug: 'objectivec' });
```

ESLint

[error] 'objectivec' is assigned a value but never used.

(no-unused-vars)